### PR TITLE
Scripts/Spells: Migrate Lightwell to Script

### PR DESCRIPTION
**Pull request by Sorikoff**
**Changes proposed:**
Migrate Lighwell to scripts system.

I have noticed that Lighwell heals caster only once (object becomes inaccessible after the first usage), but the same behaviour has been present before this change.

**Target branch(es):** 3.3.5

**Tests performed:** Works as it did.

### CODE REVIEW

File: sql/updates/world/3.3.5/9999_99_99_99_world.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_pri_lightwell');
+INSERT INTO `spell_script_names` VALUES
+(60123,'spell_pri_lightwell');
```
Comment (Aokromes) [inline sql/updates/world/3.3.5/9999_99_99_99_world.sql]: missing newline.

File: src/server/scripts/Spells/spell_priest.cpp [hunk 2]
```diff
@@ -678,6 +680,48 @@ class spell_pri_item_t6_trinket : public SpellScriptLoader
...
+    void HandleScriptEffect(SpellEffIndex /* effIndex */)
+    {
+        Creature* caster = GetCaster()->ToCreature();
+        if (!caster || !caster->IsSummon())
+            return;
+
+        uint32 spellId = 0;
+        switch (caster->GetEntry())
+        {
+            case 31897: spellId = 7001; break;
```
Comment (Keader) [inline src/server/scripts/Spells/spell_priest.cpp]: why not transform those magic numbers in enum?

Comment (Sorikoff) [reply src/server/scripts/Spells/spell_priest.cpp]: They all indicate the same (Lightwell/Lightwell Renew), and the only difference is level. I am unsure if it is needed to enum each Lightwell1, Lightwell2, and so on.

Comment (Faq) [reply src/server/scripts/Spells/spell_priest.cpp]: Lightwell_LightwellRenew = 7001 ?

Comment (Sorikoff) [reply src/server/scripts/Spells/spell_priest.cpp]: Updated PR. Thanks!

**Comment by jackpoz** [comment]
Thanks for the PR :)

**Pull request merged**